common-react/@dbeaver/ui-kit/src/Radio/RadioGroup.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+
+import { useId } from 'react';
+import './RadioGroup.css';
+import { _RadioGroup, _RadioProvider, Radio, type RadioProviderProps } from './index.js';
+import type { ControlSize } from '../types/controls.js';
+
+type ConditionalRadioProps =
+  | {
+      items: Array<{ value: string; label: string }>;
```

Comment:
should we use the same pattern as in the selector to extract value/label from any object?

Reply:
Actually I just added this option with items prop for more flexibility, I would not recommend or promote this approach in any way, so we can just remove those props, but I would not make it more complicated.

Reply:
If so, we really can remove it